cluster.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: cluster.x-k8s.io/v1alpha3
```

Comment:
I guess these 2 files have been committed by accident

Reply:
You're right! Thanks

Reply:
I removed them. FYI: The secret.yaml only contained dummy data.